actions/deploy-lambda/action.yaml

```diff
@@ -42,29 +46,9 @@ runs:
           role-duration-seconds: ${{ steps.app_settings.outputs.aws_role_duration }}
           aws-region: ${{ steps.app_settings.outputs.aws_region }}
           audience: ${{ steps.app_settings.outputs.aws_audience }}
-
-      - name: Update container lambda - get image uri with given image tag
+          
+      - name: Deploy lambda
         if: ${{ steps.app_settings.outputs.app_registry == 'ecr' }}
```

Comment:
this step is needed to deploy lambda container

Reply:
Ok, then I need to distinguish the serverless deployment path. I am going to add an app parameter say `deployer: sls` in the `.cicd-profile.yaml` for this purpose. I would make sure that if some one not defines that param in their project then it would take some default value